include/curl/easy.h

```diff
@@ -25,6 +25,49 @@
 extern "C" {
 #endif
 
+/* Application can use a memory PKCS12 certificate with CURLOPT_SSLCERT
+ *  with OpenSSL, Schannel or SecTransport
+ *
+ *
+ *  Example if with certificate binary data and size in:
+ *    void* certdata;
+ *    size_t certsize;
+ *    ...
+ *  set the certificate with:
+ *    struct curl_blob structblob;
+ *    structblob.data = certdata;
+ *    structblob.len = certize;
+ *    structblob.flags = CURL_BLOB_DUPFLAG_COPY;
+ *    curl_easy_setopt(curl, CURLOPT_SSLCERT_BLOB, &structblob);
+ *  struct curl_blob is just a structures which contain pointer to
+ *  data buffer, length and a boolean flag about buffer duplication
+ *
+ *  with flags CURL_BLOB_DUPFLAG_COPY, certdata can be discarded
+ *    after calling curl_easy_setopt
+ *  with CURL_BLOB_DUPFLAG_NOCOPY, the data pointed to is NOT copied by
+ *     the library: as a consequence, it must be preserved by the calling
+ *     application until the associated transfer finishes
+ *
+ * Note : never call curl_init_blob_xxx without a pointer to a
+ *   valid curl_blob
+ *
+ *
+ * about the idea, see https://curl.haxx.se/mail/lib-2016-09/0074.html */
+
+
+#define CURL_BLOB_DUPFLAG_COPY     (1)
+#define CURL_BLOB_DUPFLAG_NOCOPY   (0)
+#define CURL_BLOB_DUPFLAG_COPYMASK (1)
+
+/* the struct curl_blob store binary data parameters
+ * the structure size is 0x20 and can be followed by binary data */
```

Comment:
The size is not necessarily 0x20 bytes. And it's not important enough to mention the size in a comment here. It's also not important if data "can be followed". Anything can follow a struct in memory...

Reply:
I remove " the structure size is 0x20 and can be followed by binary data"